hub/util/get_storage_provider.py

```diff
@@ -42,10 +42,10 @@ def storage_provider_from_path(
         token (str): token for authentication into activeloop
 
     Returns:
-        If given a valid S3 path i.e starts with s3:// returns the S3Provider and mode. (credentials should either be in creds or the environment)
-        If given a path starting with mem://return the MemoryProvider and mode.
-        If given a valid local path, return the LocalProvider and mode.
```

Comment:
This method doesn't return `mode`, so it removed all mention of it. I think this is just an out-of-date docstring, please correct me if I'm wrong.